examples/app.py

```diff
@@ -169,17 +176,19 @@ def create_object(bucket, file_name, stream):
     provider = RecordIdProvider.create(object_type='rec', object_uuid=rec_uuid)
     data = {
         'pid_value': provider.pid.pid_value,
-        'files': [
+        '_files': [
```

Comment:
@JavierDelgadoFernandez Note that @sourabhlal is updating this section to use Invenio-Records-Files. @sourabhlal any updates on when you'll updated PR #40